docs/output.md

```diff
@@ -180,7 +182,7 @@ When `--remove_ribo_rna` is specified, the pipeline uses [SortMeRNA](https://git
 
 ## Alignment and quantification
 
-### STAR and Salmon
+### STAR, Salmon and Kallisto
```

Comment:
This is the alignment section so we don't need to refer to Kallisto here as it can't quantify from BAMs whereas Salmon can.

Reply:
It's the alignment AND quantification section. There is a passage here which refers to the pseudoaligner-only step, so Kallisto belongs I think.